presto-ui/src/index.jsx

```diff
@@ -6,6 +6,10 @@ import { PageTitle } from "./components/PageTitle";
 const ClusterHUD = lazy('ClusterHUD');
 const QueryList = lazy('QueryList');
 
+jQuery.htmlPrefilter = function( html ) {
+    return html;
+};
+
```

Comment:
What's the purpose of this function?

Reply:
According to https://api.jquery.com/jQuery.htmlPrefilter/ since 3.5.0 this function by default is already what we set here

> jQuery leaves input unmodified since 3.5.0. Older versions would greedily ensure that all tags were XHTML-compliant. This included anything that looked like an HTML tag, but was actually within a string (e.g. <a title="<div />"><>), leading to potential security issues. For more information, see the [jQuery Core 3.5 Upgrade guide](https://jquery.com/upgrade-guide/3.5/).

Reply:
we can remove it and see if there is any regression. I can comment it out and remove it completely after 1 or 2 releases.

Reply:
done! I did a quick walkthrough of most of the UI components. Everything works well.

Reply:
IMO I don't think we need to comment it out. Leaving the comment won't do much for us since it's basically dead code. If there's a weird regression in the UI, you can always try to find the offending commit via `git bisect` and just revert it. One option for this PR is to split it into two commits, one to remove the prefilter, and the rest for the img/style-src directives so that the changes for the icons doesn't get tangled with the prefilter change in the case of a regression.

Reply:
let me just remove it. hehe